sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -192,10 +193,12 @@ private static File findTsConfig(InputFile inputFile, File projectBaseDir) {
 
   }
 
-  private static void setNodePath(File typescriptLocation, ProcessBuilder processBuilder) {
-    Map<String, String> environment = processBuilder.environment();
-    LOG.debug("Setting 'NODE_PATH' to " + typescriptLocation);
-    environment.put("NODE_PATH", typescriptLocation.getAbsolutePath());
+  private static void setNodePath(@Nullable File localTypescript, ProcessBuilder processBuilder) {
+    if (localTypescript != null) {
+      Map<String, String> environment = processBuilder.environment();
+      LOG.debug("Setting 'NODE_PATH' to " + localTypescript);
+      environment.put("NODE_PATH", localTypescript.getAbsolutePath() + java.io.File.pathSeparator + environment.getOrDefault("NODE_PATH", ""));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this fully qualified name with "File" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1942)